django_project/lesson/views/worksheet.py

```diff
@@ -257,3 +260,48 @@ def post(self, request, *args, **kwargs):
         section = Section.objects.get(slug=kwargs.get('section_slug'))
         worksheets = Worksheet.objects.filter(section=section)
         return re_order_features(request, worksheets)
+
+
+
+class WorksheetModuleQuestionAnswers(WorksheetMixin, SectionMixin, DetailView):
+    """Show correct answers to module questions.
+
+    :param request: HttpRequest object
+    :type request: HttpRequest
+    """
+    # headline = _(u'Worksheet answers to module questions')
+    context_object_name = 'worksheets'
+    template_name = 'worksheet/question_answers.html'
+
+    def get_context_data(self, **kwargs):
+        """Create context for use in the templates."""
+
+        context = super(WorksheetModuleQuestionAnswers, self).get_context_data(
+         **kwargs)
```

Comment:
__[Flake8]__

[E121] continuation line under-indented for hanging indent


<!-- Comment by [SideCI](https://sideci.com) -->